webpack-test/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "webpack-test",
+  "version": "1.0.0",
+  "description": "MMPavlov",
+  "main": "index.js",
+  "dependencies": {
+    "@babel/preset-env": "^7.15.6",
+    "@babel/preset-react": "^7.14.5",
+    "babel": "^6.23.0",
+    "babel-loader": "^8.2.2",
+    "html-webpack-plugin": "^5.3.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "webpack": "^5.53.0",
+    "webpack-cli": "^4.8.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.15.7",
+    "@babel/core": "^7.15.5",
+    "webpack-dev-server": "^4.2.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:dev": "webpack serve --config ./webpack.config.dev.js",
```

Comment:
According to the task description you need to use env variables here. It can be [mode](https://webpack.js.org/configuration/mode/) or [cross-env](https://www.npmjs.com/package/cross-env)

Reply:
Fixed